api/database/migrations/2022_08_02_175856_add_citizen_veteran_fields.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddCitizenVeteranFields extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('citizenship')->nullable();
+            $table->boolean('veteran')->nullable();
```

Comment:
By convention, we've been naming boolean columns with a "is_" prefix.  For example, is_gov_employee, is_woman, is_indigenous.

Reply:
done https://github.com/GCTC-NTGC/gc-digital-talent/pull/3564/commits/9c5fc787b95b250d414583691b720713a72677ab